### PR TITLE
Add resources link to relevant k8s pages

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Added resources link to /kubernetes/what-is-kubernetes([doc](https://docs.google.com/document/d/19ZAsyujRHjOBIKP15F-4Z9L24KpfoYrx0YpL-0gO6sE/edit#)) and /kubernetes/compare([doc](https://docs.google.com/document/d/1JngYmRhohPo62yU72V1xpzCCILI21SmHKKttBxHjPog/edit#))

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: 
    - https://ubuntu-com-11567.demos.haus/kubernetes/compare
    - https://ubuntu-com-11567.demos.haus/kubernetes/what-is-kubernetes
- See that requested changes have been made
- Test resources link

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5225 https://github.com/canonical-web-and-design/web-squad/issues/5224

### CODE REVIEW

File: templates/kubernetes/compare.html [hunk 1]
```diff
@@ -425,6 +426,7 @@ <h4>Case study</h4>
         </div>
       </div>
       <p class="p-heading--4"><a href="/engage/atresmedia-charmed-kubernetes-case-study">Atresmedia dominates Spain's OTT media market with Charmed Kubernetes</a></p>
+      <p><a class="p-link--inverted" href="/kubernetes/resources">Find more resources&nbsp;&rsaquo;</a></p>
```
Comment (petesfrench) [inline templates/kubernetes/compare.html:429]:
```suggestion
      <p><a href="/kubernetes/resources">Find more resources&nbsp;&rsaquo;</a></p>
```
The background is not suru so `p-link--inverted` is not needed

File: templates/kubernetes/what-is-kubernetes.html [hunk 2]
```diff
@@ -265,6 +265,7 @@ <h3 class="p-heading--4">Video</h3>
             <h3 class="p-heading--4">Whitepaper</h3>
           </div>
           <p><a class="p-link--inverted" href="https://ubuntu.com/engage/kubernetes-deployment-enterprise-whitepaper">Five strategies to accelerate Kubernetes deployment in the enterprise</a></h3>
```
Comment (petesfrench) [inline templates/kubernetes/what-is-kubernetes.html:267]:
```suggestion
          <p><a class="p-link--inverted" href="/engage/kubernetes-deployment-enterprise-whitepaper">Five strategies to accelerate Kubernetes deployment in the enterprise</a></h3>
```

**Review by petesfrench — COMMENTED** [review]
Looks good, just a couple small changes. Also, the resources link is a slightly different shade of grey to the link above. Not sure if this was intentional as I haven't seen this pattern elsewhere:
image

**Comment by mtruj013** [comment]
Thanks for reviewing @petesfrench ! The text color thing was not intentional, but I also don't really know what's happening there. When I inspect the elements both show up as having the same color 🤔

**Review by petesfrench — APPROVED** [review]
LGTM

**Pull request merged**